src/sage/coding/databases.py

```diff
@@ -7,6 +7,11 @@
 
 lazy_import('sage.libs.gap.libgap', 'libgap')
 
+# Import the following function so that it is available as
+# sage.codes.databases.self_dual_binary_codes sage.codes.databases functions
+# somewhat like a catalog in this respect.
+lazy_import('sage.coding.self_dual_codes', 'self_dual_binary_codes')
+
```

Comment:
```
# Import the following function so that it is available as
# sage.coding.databases.self_dual_binary_codes and  codes.databases.self_dual_binary_codes
# somewhat like a catalog in this respect.
```

Reply:
Thanks, I've made some related changes

Reply:
There is no `sage.codes`. It is `sage.coding`.

Reply:
Thanks.